app/src/docs/v1.api-spec.yaml

```diff
@@ -3663,6 +4076,47 @@ components:
       example: 3cb9acc7-cfd8-4491-b091-1277bc0ec303
       required: true
   schemas:
+    AdminExternalApi:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: ID of ExternalApi.
+          example: 1223h78b-21d2-4128-b4e8-02ad3daipoi32
+        formId:
+          type: string
+          format: uuid
+          description: Form ID this ExternalApi belongs to.
+          example: 1223h78b-21d2-4128-b4e8-02ad3daipoi32
+        name:
+          type: string
+          description: Name of the ExternalApi - unique per form.
+          example: BC Registries Number Lookup
+        ministry:
+          type: string
+          description: Ministry Name of Form
+          example: CITZ
+        formName:
+          type: string
+          description: Name of the ExternalApi - unique per form.
+          example: My Form
+        endpointUrl:
+          type: string
+          format: url
+          description: Name of the ExternalApi - unique per form.
```

Comment:
Maybe change to "URL endpoint used for the external api"?

Reply:
Good catch, thanks. Fixing.